examples/sql-integration/seeds/db_out/init.sql

```diff
@@ -1,5 +1,4 @@
 CREATE TABLE results (
     id SERIAL PRIMARY KEY,
-    joined INT,
-    total INT
+    joined INT
```

Comment:
With this database, the area code can only be reconstructed implicitly from the serial primary key. Ideally, the garble program would return a `[(u8, u16); 10]` where the first element of the tuple is set to the area code (i.e. `[(0, 0), (1, 0), (2, 0),...]` initially before the join). Then the output query in the policy can insert the tuple into different columns for the area code and number of people with disabilities for this area code.
However, I'm unsure if this would increase the circuit size. Ideally it should not, as we set the first element to public constants.

Reply:
If it is not easily changeable, we can do it at a later time if necessary. But it would be nice to know if it is possible without overhead.